gstreamer-sdp/src/s_d_p_media.rs

```diff
@@ -118,27 +128,27 @@ impl SDPMedia {
         }
     }
 
-    pub fn get_attribute(&self, idx: u32) -> Option<SDPAttribute> {
+    pub fn get_attribute(&self, idx: u32) -> Option<&SDPAttribute> {
         unsafe {
-            from_glib_none(ffi::gst_sdp_media_get_attribute(self.to_glib_none().0, idx) )
+            &*(from_glib_none(ffi::gst_sdp_media_get_attribute(self.to_glib_none().0, idx)) as *mut SDPAttribute)
         }
     }
 
-    pub fn get_attribute_val(&self, key: &str) -> Option<String> {
+    pub fn get_attribute_val(&self, key: &str) -> Option<&str> {
         unsafe {
             from_glib_none(ffi::gst_sdp_media_get_attribute_val(self.to_glib_none().0, key.to_glib_none().0))
         }
     }
 
-    pub fn get_attribute_val_n(&self, key: &str, nth: u32) -> Option<String> {
+    pub fn get_attribute_val_n(&self, key: &str, nth: u32) -> Option<&str> {
         unsafe {
             from_glib_none(ffi::gst_sdp_media_get_attribute_val_n(self.to_glib_none().0, key.to_glib_none().0, nth))
         }
     }
 
-    pub fn get_bandwidth(&self, idx: u32) -> Option<SDPBandwidth> {
+    pub fn get_bandwidth(&self, idx: u32) -> Option<&SDPBandwidth> {
         unsafe {
-            from_glib_none(ffi::gst_sdp_media_get_bandwidth(self.to_glib_none().0, idx))
+            &*(from_glib_none(ffi::gst_sdp_media_get_bandwidth(self.to_glib_none().0, idx)) as *mut SDPBandwidth)
```

Comment:
For all the other references, don't use `from_glib_none` either. This one here for example would be
```Rust
let ptr = ffi::gst_sdp_media_get_bandwidth(self.to_glib_none().0, idx);
if ptr.is_null() {
    None
} else {
    Some(&*(ptr as *mut SDPBandwidth))
}
```

(or maybe `as *const ...`)